lib/msf/core/feature_manager.rb

```diff
@@ -124,6 +125,13 @@ class FeatureManager
         requires_restart: false,
         default_value: true,
         developer_notes: 'Enabled in Metasploit 6.4.x'
+      }.freeze,
+      {
+        name: SHOW_AUTOCOMPLETION_PROMPT,
+        description: 'When enabled, an autocompletion prompt will be shown when typing in commands',
+        requires_restart: true,
```

Comment:
Maybe this could be `false` if we update `def set(name, value)` to have a set-effect like it does for wrapped tables